www/artworks/new.php

```diff
@@ -0,0 +1,144 @@
+<?php /** @noinspection PhpUndefinedMethodInspection,PhpIncludeInspection */
+require_once('Core.php');
+
+session_start();
+
+$exception = $_SESSION['exception'] ?? null;
+
+if ($exception) {
+	http_response_code(422);
+	session_unset();
+}
+
+$success = $_SESSION['successfully-submitted-artwork'] ?? null;
+
+if ($success) {
+	http_response_code(201);
+	session_unset();
+}
+
+?>
+<?= Template::Header(
+	[
+		'title' => 'Submit Artwork',
+		'highlight' => '',
+		'description' => 'Submit public domain artwork to the database for use as cover art.'
+	]
+) ?>
+<main>
+	<section class="narrow">
+		<hgroup>
+			<h1>Submit Artwork</h1>
+			<h2></h2>
+		</hgroup>
+
+		<?= Template::Error(['exception' => $exception]) ?>
+
+		<? if ($success) { ?>
+			<p class="message success">
+				Thank you for submitting artwork. It will be reviewed and added to the database if it is approved.
+			</p>
+		<? } ?>
+
+		<form method="post" action="/artworks" enctype="multipart/form-data">
+			<fieldset>
+				<legend>Artist Details</legend>
+				<div>
+					<label>
+						Artist Name
+						<datalist id="artist-names">
+							<?php foreach (Artist::GetAll() as $artist): ?>
+								<option value="<?= $artist->Name ?>"></option>
+							<?php endforeach; ?>
+						</datalist>
+						<input type="text" name="artist-name" list="artist-names" required="required"/>
+					</label>
+					<label>
+						Year of Death
+						<input type="number" name="artist-year-of-death" min="0" max="<?= gmdate('Y') ?>"/>
+					</label>
+				</div>
+			</fieldset>
+			<fieldset>
+				<legend>Artwork Details</legend>
+				<div>
+					<label>
+						Artwork Name
+						<input type="text" name="artwork-name" required="required"/>
+					</label>
+					<label for="artwork-year">
+						Year of Completion
+						<label>
+							(circa?
+							<input type="checkbox" name="artwork-year-is-circa"/>)
+						</label>
+						<input type="number" id="artwork-year" name="artwork-year" min="0" max="<?= gmdate('Y') ?>"/>
+					</label>
+				</div>
+				<label>
+					Artwork Tags
+					<input type="text" name="artwork-tags" placeholder="tags, comma-separated"/>
+				</label>
+			</fieldset>
+			<fieldset id="pd-proof">
+				<legend>Proof of Public Domain Status</legend>
```

Comment:
I think you noted this, too, but we should give some guidance to the form submitter here. Maybe as a short paragraph, maybe as some tooltips. (We can do this in a subsequent PR.)

Reply:
Agreed, let's leave this for later once the functionality of the form is complete.